modules/accumulo/src/main/java/io/fluo/recipes/accumulo/export/DifferenceExport.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2016 Fluo authors (see AUTHORS)
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.fluo.recipes.accumulo.export;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+import io.fluo.api.data.Bytes;
+import io.fluo.api.data.Column;
+import io.fluo.api.data.RowColumn;
+import org.apache.accumulo.core.data.Mutation;
+
+public abstract class DifferenceExport<K, V> implements AccumuloExport<K> {
+
+  private V oldVal;
+  private V newVal;
+
+  public DifferenceExport() {}
+
+  public DifferenceExport(V oldVal, V newVal) {
+    this.oldVal = oldVal;
+    this.newVal = newVal;
+  }
+
+  protected abstract Map<RowColumn, Bytes> generateData(K key, V val);
```

Comment:
Added javadoc in 68fc74e9404ff3.  Full documentation can be done as separate PR.